src/functions-reference/functions_index.qmd

```diff
@@ -294,6 +294,21 @@ pagetitle: Alphabetical Index
  - <div class='index-container'>[distribution statement](unbounded_discrete_distributions.qmd#index-entry-0c7465aa1beceb6e7e303af36b60e2b847fc562a) <span class='detail'>(unbounded_discrete_distributions.html)</span></div>
 
 
+<a id='beta_neg_binomial_cdf' href='#beta_neg_binomial_cdf' class='anchored unlink'>**beta_neg_binomial_cdf**:</a>
```

Comment:
Why is this PR touching negative binomial?  Are you up to date with the main branch?

Reply:
hmmm... that's odd. Let me check.

Reply:
As far as I can tell, I am up to date with the main branch, so I'm not sure how this error occurred. I suppose I could simply delete the extra lines.